### PR TITLE
Add briefing info tab on project tasks

### PR DESCRIPTION
**Pull request by dreispt**

### CODE REVIEW

File: project_task_briefing/README.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
```
Comment (yvaucher) [inline project_task_briefing/README.rst:3]: A title would be great

File: project_task_briefing/README.rst [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+This module places the Description field in a new initial tab titled 
+"Briefing": it is expected to be used as an initial the description of
+the work to be done.
+
+Roadmap
+=======
+
+* Add a "debriefing" tab: it would be useful to also have a debriefing field
+  to report on the work done and results.
+
```
Comment (yvaucher) [inline project_task_briefing/README.rst:14]: Bugtracker to summon here

**Comment by yvaucher** [comment]
just readme template that changed in the mean time otherwise :+1:

**Comment by dcorio** [comment]
:-1: 
1) if you don't enable work activities (Settings > Project > Log work activities on tasks) you get an empty and useless Description tab.
2) having both Briefing and Description tabs can be quite misleading

to achieve what you propose, i would suggest to move the work activities (work_ids) to a dedicated tab.

**Comment by rafaelbn** [comment]
Hi, no runbot to test it and give an opinion

**Pull request closed without merging**